### PR TITLE
Fix `portfolio/po` plots not showing in the OpenBB Docs

### PR DESCRIPTION
**Pull request by JerBouma**
Woops:

image

Fixed now:

image

### CODE REVIEW

File: website/content/terminal/guides/intros/portfolio/po.md [hunk 1]
```diff
@@ -377,7 +377,7 @@ Further analysis can be done by looking at the portfolio's returns <a href="http
 
 ````
 2022 May 31, 03:39 (🦋) /portfolio/po/ $ plot HRP_0 -ct SECTOR -hi -dd
-```
+````
```
Comment (jmaslek) [inline website/content/terminal/guides/intros/portfolio/po.md:381]: Shouldn't they all be 3 ticks?

Comment (JerBouma) [reply website/content/terminal/guides/intros/portfolio/po.md:381]: Whether it's 3 or 4 doesn't really matter, just needs to be consistent which it wasn't.

**Review by jmaslek — APPROVED** [review]

**Comment by JerBouma** [comment]
Yay

**Comment by abuoeissa9** [comment]
rth1090

**Pull request merged**